.eslintrc.json

```diff
@@ -1,6 +1,17 @@
 {
-  "extends": "next/core-web-vitals",
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "next/core-web-vitals",
+    "plugin:jsx-a11y/recommended",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": ["@typescript-eslint", "prettier"],
   "rules": {
-    "@next/next/no-img-element": "off"
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unnecessary-type-constraint": "off",
+    "@typescript-eslint/ban-types": ["off", { "types": ["{}", "object"] }],
+    "@next/next/no-img-element": "off",
+    "prettier/prettier": "error"
```

Comment:
What is the reason these exceptions are here? Not really against them, just curious

Reply:
It requires some work to be able to remove these as they raise errors with the existing code base

Reply:
I think some of these could be removed (for example no-unused-vars) with a bit of work in another PR since this one is quite big already with style changes